KaiSDK/Kai.py

```diff
@@ -0,0 +1,29 @@
+from KaiSDK.EventManager import EventManager
+
+
+class Kai:
+    def __init__(self, id = 0, capabilities = 0, hand = 0):
+        self.id = 0
```

Comment:
Don't initialize it to 0. Otherwise DefaultKai, DefaultLeftKai and DefaultRightKai will all have the same Kai ID

Reply:
Fixed